public/src/components/CenterMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+
+export enum KnownMessage {
+    No_Submission = "Select a submission from the results table",
+}
+
+// CenteredMessage is used by Review Assignments and View results to display a message in the center
+export const CenteredMessage = (message: string | KnownMessage) => {
```

Comment:
Could this be written as a component such that we can call it by doing:

```js
<CenteredMessage message="some message" />
```

Not sure what difference it makes compared to the current:

```js
CenteredMessage("some message")
```

I prefer the first one, although we do have the latter one scattered about our codebase as well.